modules/oe_content_publication/config/install/field.storage.node.oe_publication_contact.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
```

Comment:
Field oe_publication_contacts from EWPP-466 has to be used

Reply:
This field is used now.